src/views/Dashboard.vue

```diff
@@ -50,7 +46,18 @@
           </v-col>
 
           <v-col cols="12" md="3" style="position: relative">
-            <!-- FUTURE: add "Liquidator or Custodian of Records" here -->
+            <section v-if="isHistorical && getCustodians.length >= 1">
```

Comment:
A means of hiding this until we actually set and use Custodians, which would only be in the event of a future historical state.